src/monitor/process_monitor.cpp

```diff
@@ -23,13 +23,19 @@
 #include <lo2s/monitor/thread_monitor.hpp>
 #include <lo2s/process_info.hpp>
 
+#include <lo2s/perf/event_collection.hpp>
 namespace lo2s
 {
 namespace monitor
 {
 
 ProcessMonitor::ProcessMonitor() : MainMonitor()
 {
+    if (perf::requested_events().events.size() == 0)
+    {
+        throw std::runtime_error("No valid metric events given");
```

Comment:
Why exactly is this an exception? I want to be able to use lo2s without the need for recording any metrics